Server/src/controllers/authenticationController.ts

```diff
@@ -64,6 +65,43 @@ export class AuthenticationController {
         }
     }
 
+    private validateUserDetailRequest(request: Request): boolean {
+        if (request.query.hasOwnProperty('password') || request.query.hasOwnProperty('organization')) {
+            return true;
+        } else
```

Comment:
in which cases could this return false?

Reply:
if the request does not have both password and organization name, then it would return false